pepper-apis/src/main/java/org/broadinstitute/ddp/model/familyhistory/familyHistoryMapping.conf

```diff
@@ -0,0 +1,96 @@
+// Sample section of study builder configuration that would provid configuration to establish
+// mapping between activity instance data and serialized family history model
+{
+  "familyHistory":
+  // Need to point out where participant gets its medical history
+    {
+      "participantMedicalHistory": {
+        "type": "ACTIVITY",
+        "activityCode": "MEDICALHISTORY"
+      },
+
+      // Father and mother have their dedicated activities in current Family History mockups
+      "relationMappings": [
+        {
+          // Relation in this case is "hard-coded
+          // Note that these should match the FamilyHistoryRelation enums
+          "relation": "FATHER",
+          "medicalHistory": {
+            "type": "ACTIVITY",
+            "activityCode": "FATHERMEDHISTORY"
+          }
+        },
+        //  Same as above
+        {
+          "relation": "MOTHER",
+          "medicalHistory": {
+            "type": "ACTIVITY",
+            "activityCode": "MOTHERMEDHISTORY"
+          }
+        },
+        // The family relation now comes from an answer to a question.
+        // In mockups, the relations for siblings come from pull-downs.
+        // Assuming here that the pull-down values match the enum names
+        // Note that each instance of the activity will result in a new member added to family
```

Comment:
Should be standardize on some kind of `name` for each relationship so that a user can keep track of their relationships informally?  Can that be something baked into the model or would this be expressed as a common field to various `activityCode`s?

Reply:
The enums above are my attempt to standardize the relation codes. We can (and probably should) have internationalized names for them (if that is what you were asking?)